api/controllers/AddMedicationController.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * AddMedicationController
+ *
+ * @description :: Server-side actions for handling incoming requests.
+ * @help        :: See https://sailsjs.com/docs/concepts/actions
+ */
+
+ const {
+  conditionReducer,
+} = require('../utils/condition.mapper')
+
+module.exports = {
+  create: function (req, res) {
+    const data = req.session.medicalReport
+    const conditionList = conditionReducer(data.conditions)
+
+    res.view('pages/medications/add', {
+      conditionList: conditionList
+    });
+  },
+
+  store: function (req, res) {
```

Comment:
Requested change: I think LGTM is also reporting this, but it's having trouble with this line and the app won't start.

```
error: Attempted to `require('/Users/jeff/code/cppd-medical-report-sails/api/controllers/AddMedicationController.js')`, but an error occurred:
--
/Users/jeff/code/cppd-medical-report-sails/api/controllers/AddMedicationController.js:13
  store: function (req, res) {
  ^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:895:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/include-all/lib/help-include-all-sync.js:293:33
    at Array.forEach (<anonymous>)
    at _recursivelyIncludeAll (/Users/jeff/code/cppd-medical-report-sails/node_modules/include-all/lib/help-include-all-sync.js:178:11)
    at includeAll (/Users/jeff/code/cppd-medical-report-sails/node_modules/include-all/lib/help-include-all-sync.js:317:5)
    at helpBuildDictionary (/Users/jeff/code/cppd-medical-report-sails/node_modules/include-all/lib/help-build-dictionary.js:46:13)
    at Function.module.exports.optional (/Users/jeff/code/cppd-medical-report-sails/node_modules/include-all/index.js:67:10)
    at loadActionModules (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/lib/app/private/controller/load-action-modules.js:27:14)
    at async.auto.controller (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/lib/app/load.js:102:9)
    at runTask (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:1619:13)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:1559:13
    at processQueue (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:1569:13)
    at taskComplete (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:1588:9)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:1612:17
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:906:16
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/lib/app/load.js:192:14
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:3861:9
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:421:16
    at replenish (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:941:25)
    at iterateeCallback (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:931:17)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:906:16
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:3858:13
    at afterwards (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/lib/app/private/loadHooks.js:351:16)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:3861:9
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:421:16
    at replenish (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:941:25)
    at iterateeCallback (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:931:17)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:906:16
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:3858:13
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:421:16
    at iteratorCallback (/Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:998:13)
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/node_modules/async/dist/async.js:906:16
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/sails/lib/app/private/loadHooks.js:241:38
    at /Users/jeff/code/cppd-medical-report-sails/node_modules/async-listener/glue.js:188:31
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
```

Reply:
hmm... I don't see this at all? Like the code at the line number quoted in the error message is different even? Maybe it got fixed when updating from master?